src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/status.html.twig

```diff
@@ -22,10 +22,8 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
-{% set isValid = record[column.id] is same as(1) %}
```

Comment:
I saw that change made recently to compare from string to int, idk why it needed to be done, but for the sake of simplicity now i made "truthy" unstrict comparison which should work for both cases

Reply:
That makes sense, apparently `same as` is the twig equivalent of `===`
Only thing that worries e is if `'0'` will correctly be casted to int and be considered falsy?
Maybe we can make this more explicit with: `{% set isActive = record[column.id]|format_number %}`

Seems like drupal has filters like `value|boolean` or `value|integer` but I guess they added their custom filters, it doesn't seem to be part of the native twig filters, too bad.